local-cli/generator-android/templates/package/MainActivity.java

```diff
@@ -24,7 +24,7 @@ protected String getMainComponentName() {
      */
     @Override
     protected boolean getUseDeveloperSupport() {
-        return BuildConfig.DEBUG;
+        return true;
```

Comment:
We can't have separate release and debug builds? I don't think we should set it to true all the time.